### PR TITLE
Response to review feedback (classes of stock, secondary market)

### PR DESCRIPTION
**Pull request by RacheltheEditor**

### CODE REVIEW

**Comment by jlevy** [comment]
Looks good but will want to get preferred and commons stock as a formal def before merging.

File: README.md [hunk 1]
```diff
@@ -835,6 +838,10 @@ Some detail on how this works:
     investors get their money back and also participate in proceeds from common stock.
   - Whether there is a **cap**, which limits the payout if it is participating.
 
+  For more information on each of these,
+  [this Medium post](https://medium.com/@CharlesYu/the-ultimate-guide-to-liquidation-preferences-478dda9f9332)
```
Comment (jlevy) [inline README.md]: Nit: Omit Medium.

Comment (jlevy) [reply README.md]: Formatting: Best not to put content both before and after a sub-bulleted section.

File: README.md [hunk 2]
```diff
@@ -954,60 +967,61 @@ Liquidity events are the usual way shareholders in private companies sell their
...
-- 🔸 While many who hold stock hope or expect they need only find a willing buyer, in
-  practice, secondary sales only work out in a few situations.
-  In some cases, you may have luck selling stock privately to an individual, like a board
-  member or former executive who wishes to increase their ownership.
-  You can find further discussion of this topic
-  [on Quora](https://www.quora.com/How-do-employees-in-startups-sell-stock-in-the-secondary-markets).
-- 🔸 Sales generally require the agreement and cooperation of the company, for both
+- 🄳 A **secondary market transaction** (or **private sale**) occurs when private company stock
+  is sold to another private party.
+  These sales are unusual because the value of a private company’s stock is uncertain, but
```
Comment (jlevy) [inline README.md]: This line now technically is misleading—they are unusual not so much because of uncertainty (lots of transactions where values are uncertain occur in markets all the time) but because of buyers and sellers finding each other and because a company may not approve.

File: README.md [hunk 3]
```diff
@@ -954,60 +967,61 @@ Liquidity events are the usual way shareholders in private companies sell their
...
+  they can sometimes occur, such as when an employee sells to an accredited investor who
+  wants to invest in the company.
+  [By contrast](https://www.investopedia.com/terms/s/secondarymarket.asp), companies sell
```
Comment (jlevy) [inline README.md]: Nit: Contrast should link to primary market, not secondary.

File: README.md [hunk 4]
```diff
@@ -954,60 +967,61 @@ Liquidity events are the usual way shareholders in private companies sell their
...
-  and [Founders Circle](http://www.founderscircle.com/). The goal of these firms is to make
-  money of these purchases, so it will work if they believe the stock has chance of
-  significant future value.
-  In other words, if your company isn’t doing well, don’t be surprised if you can’t sell at
-  all on the secondary market.
+  [in recent years](https://techcrunch.com/2015/10/14/selling-private-company-shares-2-0/),
+  and more companies are seeing value in offering opportunities for sale or private equity
+  to [**secondary buyers**](https://en.wikipedia.org/wiki/Private_equity_secondary_market).
+  - Startups in late-stage funding rounds might offer employees the opportunity to sell
+    private equity in “secondaries,” as they’re are called in Silicon Valley, where secondary
```
Comment (jlevy) [inline README.md]: Article is a good addition. But this quoted term and "as they're called in Silicon Valley" have the confusing effect of making it sound like "secondaries" and "secondary market transactions" are different things. They are the same.

File: README.md [hunk 5]
```diff
@@ -954,60 +967,61 @@ Liquidity events are the usual way shareholders in private companies sell their
...
-  [Akkadian Ventures](https://www.akkadianventures.com/),
-  [Industry Ventures](http://www.industryventures.com/), [Atlas Peak](http://www.atlaspeakcap.com/),
-  and [Founders Circle](http://www.founderscircle.com/). The goal of these firms is to make
-  money of these purchases, so it will work if they believe the stock has chance of
-  significant future value.
-  In other words, if your company isn’t doing well, don’t be surprised if you can’t sell at
-  all on the secondary market.
+  [in recent years](https://techcrunch.com/2015/10/14/selling-private-company-shares-2-0/),
+  and more companies are seeing value in offering opportunities for sale or private equity
+  to [**secondary buyers**](https://en.wikipedia.org/wiki/Private_equity_secondary_market).
```
Comment (jlevy) [inline README.md]: FYI this wiki link is actually referring to private equity investments, a very different thing. Will revert.

File: README.md [hunk 6]
```diff
@@ -954,60 +967,61 @@ Liquidity events are the usual way shareholders in private companies sell their
...
+  - Startups in late-stage funding rounds might offer employees the opportunity to sell
+    private equity in “secondaries,” as they’re are called in Silicon Valley, where secondary
+    sales are
+    [becoming more popular](https://medium.com/@rizstanford/secondary-sales-in-vc-backed-startups-a-quick-primer-for-entrepreneurs-bdc25ea7f39a).
+  - [SharesPost](http://sharespost.com/), [Equidate](https://www.equidateinc.com/), and
+    [EquityZen](https://equityzen.com/) have sought to establish a market around secondary sales,
+    particularly for well-known pre-IPO companies.
+    A few other
+    [secondary firms](https://www.forbes.com/sites/jeffgolman/2014/05/20/exit-strategy-why-secondary-deals-are-becoming-first-choice/#2384982339ea)
```
Comment (jlevy) [inline README.md]: Ditto.

**Comment by jlevy** [comment]
All edits above now made in #121
